code/modules/antagonists/eldritch_cult/eldritch_antag.dm

```diff
@@ -0,0 +1,270 @@
+#define RUST_CULTIST 0
+#define FLESH_CULTIST 1
+#define ASH_CULTIST 2
+
+
+/proc/isecultist(mob/living/M)
+	return M && M.mind && M.mind.has_antag_datum(/datum/antagonist/ecult)
```

Comment:
This doesn't really need to be a proc, it can even be a macro:
`#define IS_E_CULTIST(mob) (mob.mind?.has_antag_datum(/datum/antagonist/ecult))`

Reply:
all other iscultist, istraitor etc are procs, so i just followed suit